VMBackup/main/ExtensionErrorCodeHelper.py

```diff
@@ -13,6 +13,7 @@ class ExtensionErrorCodeEnum():
     #error_upload_status_blob = 16
     error = 2
     FailedRetryableSnapshotFailedNoNetwork = 76
+    FailedSnapshotLimitReached = 85
```

Comment:
Without this, why was exception not getting thrown in earlier code where this was getting assigned in freezesnapshotter.py ?